docs/src/data/mockServices.tsx

```diff
@@ -0,0 +1,78 @@
+let signUpPassword = '';
+const verifiedUsers = new Map();
+const FIXED_USERNAME = 'test';
+const FIXED_PASSWORD = 'password';
+
+export const mockServices = {
+  async getAmplifyConfig() {
+    return {};
+  },
+  async handleSignUp(formData) {
+    signUpPassword = formData.password;
+    return {};
+  },
+  async checkVerifiedContact() {
+    return { verified: {}, unverified: {} };
+  },
+  async handleSignIn({
+    username,
+    password,
+  }: {
+    username: string;
+    password: string;
+  }) {
+    if (
+      verifiedUsers.get(username).password === password ||
+      (username === FIXED_USERNAME && password === FIXED_PASSWORD)
+    ) {
+      return Promise.resolve(verifiedUsers.get(username));
+    } else {
+      return Promise.reject(new Error('Invalid username or password'));
+    }
+  },
+  async handleConfirmSignUp({
+    username,
+    code,
+  }: {
+    username: string;
+    code: string;
+  }) {
+    if (code === '123456') {
```

Comment:
This is a magic number, and probably should be added to the top of the file. With the FIXED_USERNAME and FIXED_PASSWORD.

Reply:
Thank you so much for pointing that out!